pegasus/test/test_hoc_routes.rb

```diff
@@ -9,6 +9,7 @@ class HocRoutesTest < Minitest::Test
       $log.level = Logger::ERROR # Pegasus spams debug logging otherwise
       @mock_session = Rack::MockSession.new(MockPegasus.new, 'studio.code.org')
       @pegasus = Rack::Test::Session.new(@mock_session)
+      Gatekeeper.set('hoc_activity_writes_disabled', value: false)
```

Comment:
In Pegasus, Gatekeeper settings persisted between tests (at least when running with `ruby -Itest`). Also got an error when trying to clear using Gatekeeper.clear as clear has [not been implemented](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/dynamic_config/adapters/json_file_adapter.rb#L69-L69) in the `JSONFileDatastoreAdapter` yet. That is gated on rack_env, which maybe on the machines we test on will be set to test?